Lib/asyncio/unix_events.py

```diff
@@ -51,8 +51,13 @@ def __init__(self, selector=None):
 
     def close(self):
         super().close()
-        for sig in list(self._signal_handlers):
-            self.remove_signal_handler(sig)
+        if not sys.is_finalizing():
+            for sig in list(self._signal_handlers):
+                self.remove_signal_handler(sig)
+        else:
+            warinigs.warn(f"Closing the loop {self} on interpreter shutdown "
+                          f"stage, signal unsubsription is disabled",
+                          stacklevel=2)
```

Comment:
I'm not sure that stacklevel=2 is correct. Maybe remove it?

For the warning category, I hesitate to use ResourceWarning and pass source=self. ResourceWarning are quiet by default. It's maybe better to not pollute stdout with such warning?